plugins/home/backstage-plugin-home-rss/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@roadiehq/backstage-plugin-home-rss",
+  "version": "0.0.1",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "github:RoadieHQ/roadie-backstage-plugins",
+    "directory": "plugins/home/backstage-plugin-home-rss"
+  },
+  "bugs": {
+    "url": "https://github.com/RoadieHQ/roadie-backstage-plugins/issues",
+    "email": "support@roadie.io"
+  },
+  "scripts": {
+    "start": "backstage-cli backend:dev",
+    "build": "backstage-cli backend:build",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^0.13.0",
+    "@backstage/config": "^1.0.0",
+    "@backstage/core-components": "^0.9.0",
+    "@backstage/core-plugin-api": "^1.0.0",
+    "@backstage/plugin-home": "^0.4.17",
+    "@material-ui/core": "^4.12.3",
+    "@material-ui/lab": "^4.0.0-alpha.60",
+    "cross-fetch": "^3.1.4",
+    "msw": "^0.38.2",
+    "react-use": "^17.3.2",
+    "winston": "^3.2.1"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1 || ^17.0.0"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.16.0",
```

Comment:
We just bumped the plugins versions to the latest, which will bump the backstage/cli to version 0.17.0 and has potential breaking changes. Could you please bump this package also to the latest ? 
```
yarn backstage-cli versions:bump
```